internal/flow/flow.go

```diff
@@ -290,20 +296,84 @@ func PushArtifact(ctx context.Context, gitSvc *git.Service, artifactFileName, re
 	return artifactSpec.ID, nil
 }
 
-func listFiles(path string) {
+// PushArtifactToReleaseManager pushes an artifact to the release manager
+func PushArtifactToReleaseManager(ctx context.Context, releaseManagerClient *httpinternal.Client, artifactFileName, resourceRoot string) (string, error) {
+	artifactSpecPath := path.Join(resourceRoot, artifactFileName)
+	artifactSpec, err := artifact.Get(artifactSpecPath)
+	if err != nil {
+		return "", errors.WithMessagef(err, "path '%s'", artifactSpecPath)
+	}
+
+	path, err := releaseManagerClient.URLWithQuery(fmt.Sprintf("artifacts/create"), url.Values{})
```

Comment:
As we don't use query parameters we can use `Client.URL` instead.

```suggestion
	path, err := releaseManagerClient.URL(fmt.Sprintf("artifacts/create"))
```

Reply:
of course 🤦‍♂️